modules/040-node-manager/openapi/values.yaml

```diff
@@ -53,9 +55,9 @@ properties:
                   effect:
                     type: string
                     enum:
-                    - NoSchedule
-                    - PreferNoSchedule
-                    - NoExecute
+                      - NoSchedule
```

Comment:
Why?

Reply:
That's how goland formats code for me. Does yor IDE change this? You can check it with Cmd+Option+L.